dist/bin/current/helperScripts/findPluginLibraries.groovy

```diff
@@ -4,37 +4,88 @@
  * Distributed under the MIT License (license terms are at https://www.github.com/eilslabs/Roddy/LICENSE.txt).
  */
 
+
```

Comment:
you extended this a lot and this is good. But as it is Groovy, we could also use the Roddy code itself to find all this or not? At least, we should investigate it.

Reply:
Right.